host/frontend/host-orchestrator/orchestrator/instancemanager.go

```diff
@@ -345,3 +391,28 @@ func (c *CVDSubcmdStartCVDServer) Run(execContext ExecContext) error {
 	cmd.Stderr = nil
 	return cmd.Run()
 }
+
+type CVDCmd interface {
+	// Runs the command and returns its combined standard output and standard error.
+	Run(execContext ExecContext, paths IMPaths, buildInfo apiv1.BuildInfo) ([]byte, error)
+}
+
+type CVDSubcmdFetchCVD struct{}
+
+func (c *CVDSubcmdFetchCVD) Run(
+	execContext ExecContext,
+	paths IMPaths,
+	buildInfo apiv1.BuildInfo) ([]byte, error) {
+	outDir := buildArtifactsDir(paths.ArtifactsRootDir, buildInfo)
+	if err := os.Setenv("ANDROID_HOST_OUT", outDir); err != nil {
+		return nil, err
+	}
```

Comment:
good point, will take it into account, however aosp/2115247 aims to avoid setting ANDROID_HOST_OUT for non relevant operations like `start` and `fetch` so this might be deleted in the future.